adminpage.css

```diff
@@ -0,0 +1,221 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Poppins", sans-serif;
+  background-color: #fff;
+  display: flex;
```

Comment:
use a wrapper container inorder to organize code better. don't use flexbox in body tag